Buildings/Experimental/DHC/Examples/Cooling/DirectUncontrolledETS.mo

```diff
@@ -0,0 +1,184 @@
+    annotation (Dialog(group="Buildings"));
+  final parameter Modelica.Units.SI.HeatFlowRate QCoo_flow_nominal[nLoa](
+    max=-Modelica.Constants.eps)={Buildings.Experimental.DHC.Loads.BaseClasses.getPeakLoad(
+    string="#Peak space cooling load",
+    filNam=Modelica.Utilities.Files.loadResource(filNam[i])) for i in 1:nLoa}
+    "Space cooling design load (<=0)";
+  final parameter Modelica.Units.SI.MassFlowRate mBui_flow_nominal[nLoa](
+    each final min=0,
+    each final start=0.5)={-QCoo_flow_nominal[i]/(cp*buiETS[i].dT_nominal) for i in 1:nLoa}
+    "Nominal mass flow rate of building cooling side";
+  Buildings.Experimental.DHC.Plants.Cooling.ElectricChillerParallel pla(
+    perChi=perChi,
+    dTApp=3,
+    perCHWPum=perCHWPum,
+    perCWPum=perCWPum,
+    mCHW_flow_nominal=mCHW_flow_nominal,
+    dpCHW_nominal=dpCHW_nominal,
+    QChi_nominal=QChi_nominal,
+    mMin_flow=0.03,
+    mCW_flow_nominal=mCW_flow_nominal,
+    dpCW_nominal=dpCW_nominal,
+    TAirInWB_nominal=298.7,
+    TCW_nominal=308.15,
+    dT_nominal=5.56,
+    TMin=288.15,
+    PFan_nominal=5000,
+    dpCooTowVal_nominal=6000,
+    dpCHWPumVal_nominal=6000,
+    dpCWPumVal_nominal=6000,
+    tWai=30,
+    dpSetPoi=dpSetPoi,
+    energyDynamics=Modelica.Fluid.Types.Dynamics.FixedInitial)
+    "District cooling plant"
+    annotation (Placement(transformation(extent={{-40,-20},{-20,0}})));
+  Buildings.BoundaryConditions.WeatherData.ReaderTMY3 weaDat(
+  final computeWetBulbTemperature=true,
+  filNam=Modelica.Utilities.Files.loadResource("modelica://Buildings/Resources/weatherdata/USA_CA_San.Francisco.Intl.AP.724940_TMY3.mos"))
+    "Weather data"
+    annotation (Placement(transformation(extent={{-80,20},{-60,40}})));
+  Modelica.Blocks.Sources.BooleanConstant on
+    "On signal of the plant"
+    annotation (Placement(transformation(extent={{-80,-20},{-60,0}})));
+  Modelica.Blocks.Sources.Constant TCHWSupSet(k=273.15+7)
+    "Chilled water supply temperature setpoint"
+    annotation (Placement(transformation(extent={{-80,-60},{-60,-40}})));
+  Buildings.Experimental.DHC.Networks.Distribution2Pipe dis(
+    redeclare final package Medium=Medium,
+    nCon=nLoa,
+    allowFlowReversal=false,
+    mDis_flow_nominal=sum(dis.mCon_flow_nominal),
+    mCon_flow_nominal=mBui_flow_nominal,
+    mEnd_flow_nominal=mBui_flow_nominal[nLoa],
+    dpDis_nominal=fill(1500,nLoa))
+    "Distribution network for district cooling system"
+    annotation (Placement(transformation(extent={{20,-20},{60,0}})));
+  Buildings.Experimental.DHC.Loads.Cooling.BuildingTimeSeriesWithETS buiETS[nLoa](
+    filNam=filNam,
+    mBui_flow_nominal=mBui_flow_nominal)
+    "Vectorized time series building load model connected with ETS for cooling."
+    annotation (Placement(transformation(extent={{30,40},{50,60}})));
+  Buildings.Fluid.Sensors.RelativePressure senRelPre(redeclare package Medium
+      =                                                                         Medium)
```

Comment:
correct the format

Reply:
Sorry about the format problem, already correct it.